src/geometry.rs

```diff
@@ -396,6 +396,14 @@ pub enum Face {
     Triangle(TriangleFace),
 }
 
+impl Face {
+    pub fn contains_vertex(&self, vertex_index: u32) -> bool {
```

Comment:
This is ok but let's then look-up all the uses of the `TriangleFace` equivalent of this (and other functions too) and think about replacing all of them with methods on this level. Generally, let's use `TriangleFace` only where other types of faces wouldn't work.

Reply:
I did. This is the first use of the any of the `contains_vertex` functions 😂 (at least of what is merged to master)

Reply:
In one of my upcoming PRs I'll do the same for other methods